.github/workflows/ci.yml

```diff
@@ -204,15 +204,27 @@ jobs:
             backend: cpp
             env: { MACOSX_DEPLOYMENT_TARGET: 10.14 }
 
+        # exclude:
+        #   # fails due to lack of a compatible compiler
+        #   - os: windows-2019
+        #     python-version: 2.7
+        #   - os: windows-2019
+        #     python-version: 3.4
+
+        #   # cpp specific test fails
+        #   - os: windows-2019
+        #     python-version: 3.5
+        #     backend: cpp
```

Comment:
```suggestion
        exclude:
          # fails due to lack of a compatible compiler
          - os: windows-2019
            python-version: 2.7
          - os: windows-2019
            python-version: 3.4

          # cpp specific test fails
          - os: windows-2019
            python-version: 3.5
            backend: cpp
```

Reply:
so that should return everything back to the point where there are no failing tests